### PR TITLE
Fix pylint invalid-name

### PR DESCRIPTION
**Pull request by jonathan-eq**

### CODE REVIEW

File: komodo/__init__.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
     try:
         from ._version import version as VERSION
     except ImportError:
-        msg = "Failed to find autogenerated _version.py."
+        msg = (
+            "Failed to find autogenerated _version.py."  # pylint: disable=invalid-name
+        )
```
Comment (berland) [inline komodo/__init__.py:11]: I would just skip the temporary variable here and just to `ImportError("Failed to ...")`

File: komodo/lint.py [hunk 2]
```diff
@@ -100,10 +102,10 @@ def lint(release_file: ReleaseFile, repository_file: RepositoryFile):
                         ),
                     ),
                 )
-        except KomodoException as e:
-            mns.append(e.error)
+        except KomodoException as komodo_exception:
+            mns.append(komodo_exception.error)
```
Comment (berland) [inline komodo/lint.py:106]: is not `mns` not an invalid name?

File: tests/data/test_releases/test_release_matrix-py38-rhel7.yml
Comment (berland) [inline tests/data/test_releases/test_release_matrix-py38-rhel7.yml:1]: not sure why this file is included in a PR that fixes invalid-name?

File: tests/test_switch.py [hunk 3]
```diff
@@ -54,5 +54,5 @@ def test_write_activator_switches_for_non_matrix_build(tmpdir):
 
     try:
         switch.create_activator_switch(Data(), prefix, release)
-    except ValueError as e:
-        pytest.fail("Unexpected ValueError " + e)
+    except ValueError as value_error:
+        pytest.fail("Unexpected ValueError " + value_error)
```
Comment (berland) [inline tests/test_switch.py:58]: maybe also rewrite to f-string while at it.

File: tests/data/test_releases/test_release_matrix-py38-rhel7.yml
Comment (jonathan-eq) [reply tests/data/test_releases/test_release_matrix-py38-rhel7.yml:1]: I guess there is one test that creates this file, and it snuck in..

**Review by berland — APPROVED** [review]
Beautiful validity. Assuming conflict resolution goes well :)

**Pull request merged**